test/feature-benchmark/scenarios_skew.py

```diff
@@ -0,0 +1,71 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+from math import floor
+from textwrap import dedent
+from typing import List
+
+from materialize.feature_benchmark.action import Action, TdAction
+from materialize.feature_benchmark.measurement_source import MeasurementSource, Td
+from materialize.feature_benchmark.scenario import Scenario
+
+
+def create_skewed_table(scale: int) -> str:
+    count = 10**scale
+    return (
+        dedent(
+            f"""
+       > CREATE TABLE skewed_table(f1 INTEGER, f2 INTEGER DEFAULT 1);
```

Comment:
From what I can gather, the `f2` fields are unnecessary for the scenario, correct? Perhaps we could drop them to avoid confusion.